w3-hw1-vy/cypress/pages/bookstore/profilePage.js

```diff
@@ -0,0 +1,74 @@
+export const profilePage = {
+    LBL_USERNAME_VALUE: "#userName-value",
+    // BTN_SUBMIT: "#submit",
+    // BTN_OK_DIALOG: "#closeSmallModal-ok",
+    BTN_GO_TO_STORE: "#gotoStore",
+    TXT_BOOK_NAME_COL: ".rt-table .rt-tr-group .rt-td:nth-child(2) a",
+    TXT_CURRENT_PAGE: ".-pageJump input",
+    TXT_TOTAL_PAGES: ".-totalPages",
+    BTN_PAGE: ".-pagination button",
+
+    isUserNameCorrect(username) {
+        cy.get(this.LBL_USERNAME_VALUE).should("have.text", username);
+        return this;
+    },
+
+    clickDeleteAllBooks() {
```

Comment:
Function name should reflect exactly what is implement inside it, you should find another name or split into smalls functions. But I prefer split due to [SOLID Design Principals](https://www.freecodecamp.org/news/solid-design-principles-in-software-development/)